Src/FluentAssertions/Equivalency/Execution/CollectionMemberOptionsDecorator.cs

```diff
@@ -20,26 +20,14 @@ public CollectionMemberOptionsDecorator(IEquivalencyOptions inner)
     }
 
     public IEnumerable<IMemberSelectionRule> SelectionRules
-    {
-        get
-        {
-            return inner.SelectionRules.Select(rule => new CollectionMemberSelectionRuleDecorator(rule)).ToArray();
-        }
-    }
+        => inner.SelectionRules.Select(rule => new CollectionMemberSelectionRuleDecorator(rule)).ToArray();
```

Comment:
🤔 Was it Qodana? If so, we should disable that rule. I don't think it's always better to use expression-bodied members.

Reply:
Yes, I get this notice from Qodana:
![image](https://github.com/fluentassertions/fluentassertions/assets/3438234/bd2ce488-b3c1-4892-841d-b45aa0a9c704)

Reply:
Let's disable it then. I'm fine if you do that in this PR.

Reply:
> Let's disable it then. I'm fine if you do that in this PR.

What do you mean, @dennisdoomen?
1. Keep the refactorings regarding expression body syntax in the PR or
2. Disable the Qodana rule as part of this PR?

If you mean the second point, I am not sure where the rule would be disabled. Can you point me in the direction?

Reply:
I prefer 2. This needs to happen in the `qodana.yml`

Reply:
I disabled the rule [`ArrangeAccessorOwnerBody`](https://www.jetbrains.com/help/rider/ArrangeAccessorOwnerBody.html) and undid the refactoring in [CollectionMemberOptionsDecorator.cs](https://github.com/fluentassertions/fluentassertions/pull/2413/files/aff14810149d3017523d0a9faf6677f9be747133#diff-32d01e9ed629abd7e7461b4bd2283efe9267baaca071a6a519d1fb358c3991cb).